application/models/meeting.go

```diff
@@ -405,7 +405,7 @@ func ConvertMeetings(ctx context.Context, meetings []Meeting, user User) ([]api.
 }
 
 // ConvertMeeting converts a model.Meeting into api.Meeting
-func ConvertMeeting(ctx context.Context, meeting Meeting, user User) (api.Meeting, error) {
+func ConvertMeeting(ctx context.Context, meeting Meeting, user User, includeParticipants bool) (api.Meeting, error) {
```

Comment:
Another technique that makes for more readable (and expandable) code would be to use an options parameter. Something like:
```go
func ConvertMeeting(ctx context.Context, meeting Meeting, user User, options ...string) (api.Meeting, error)
```
called like:
```go
_, _ = ConvertMeeting(ctx, meeting, user, "includeParticipants")
```

Reply:
True, or use constants for `IncludeParticipants` and `ExcludeParticipants` so it reads clear and you don't have to remember which is `true` or `false`